.circleci/config.yml

```diff
@@ -0,0 +1,162 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/merchant-center-application-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node_10:
+    docker:
+      - image: circleci/node:10.15
+    working_directory: *working_directory
+
+orbs:
+  cypress: cypress-io/cypress@1
+
+commands:
+  install_dependencies:
+    description: 'Installing dependencies'
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run: yarn install --pure-lockfile
+      - save_cache: *save_yarn_cache
+  build_artifacts:
+    description: 'Building artefacts'
+    steps:
+      - run: yarn build
```

Comment:
We need to persist to workspace right?

Reply:
It doesn't seem that easy. I tried for a long time to have this working. The problem is the following (as I tried to document with comments):

1. The Cypress orb uses the `~/` as its root
2. We use `~/merchant-center-application-kit`
    - We can not use `~/` cause CircleCI then fails to checkout code and they recommend using `~/<projectName>`

This leaves us with a bit of problems as we want to save from `~/merchant-center-application-kit` post build to `~/` for Cypress to be able to pick it up. I tried to persist `packages` to both `/~` and `~/merchant-center-application-kit` however still it wasn't picked up correctly. I can commit c change again and see if you spot the error.